foundry/foundry.py

```diff
@@ -857,6 +857,53 @@ def toTorch(self, raw=None, split=None):
             return FoundryDataset_Torch(inputs, targets)
         else:
             raise NotImplementedError
+
+    def toTensorflow(self, raw=None, split=None):
+        """Convert Foundry Dataset to a Tensorflow Sequence
+
+        Arguments:
+            raw (dict): The output of running ``f.load_data(as_hdf5=False)``
+                    Recommended that this is left as ``None``
+                    **Default:** ``None``
+            split (string): Split to create Tensorflow Sequence on.
```

Comment:
Consider using type annotations instead of/in addition to typing in the docstring. Might as well start doing that now as it's a feature that is only getting more useful as more Python libraries adopt typing.

Reply:
We should do this in another PR for all of the functions in the Foundry Package

Reply:
Agreed, but let's start now too